framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -517,18 +524,40 @@ protected boolean visitTypeArgs(
             : AnnotationUtils.getElementValueArray(
                 covariantAnno, covariantValueElement, Integer.class);
 
-    for (int i = 0; i < supertypeTypeArgs.size(); i++) {
-      final AnnotatedTypeMirror superTypeArg = supertypeTypeArgs.get(i);
-      final AnnotatedTypeMirror subTypeArg = subtypeTypeArgs.get(i);
-      final boolean covariant = covariantArgIndexes != null && covariantArgIndexes.contains(i);
-
-      boolean result = isContainedBy(subTypeArg, superTypeArg, covariant);
+    // JLS: 4.10.2. Subtyping among Class and Interface Types
+    // 3th paragraph
```

Comment:
The third paragraph is about comparisons between declarations (generic types).
The fourth paragraph is about comparisons between uses (parameterized types).
Should this refer to the fourth paragraph rather than the third?